deploy/operator.yaml

```diff
@@ -30,4 +30,4 @@ spec:
                 fieldRef:
                   fieldPath: metadata.name
             - name: OPERATOR_NAME
-              value: "activemq-artemis-operator"
+              value: "activemq-artemis-operator"
```

Comment:
Can't see end of file, but please make sure all files end with a line break

Reply:
fixed